google/cloud/aiplatform/jobs.py

```diff
@@ -1403,6 +1406,29 @@ def from_local_script(
                         {"name": key, "value": value}
                         for key, value in environment_variables.items()
                     ]
+            else:
+                command = [
+                    "sh",
+                    "-c",
+                    "\npip3 install -q --user --upgrade --no-warn-script-location gsutil"
+                    + f"\ngsutil -q cp {package_gcs_uri} ."
+                    + f"\npip3 install -q --user {package_gcs_uri[len(staging_bucket)+1:]}"
+                    + f"\npython3 -m {python_packager.module_name}",
+                ]
+
+                spec["container_spec"] = {
+                    "image_uri": container_uri,
+                    "command": command,
+                }
+
+                if args:
```

Comment:
Reuse `args` and `environment_variables` setting logic shared with the pre-built container condition.

Reply:
They have different spec names (`spec['container_spec']` vs `spec['python_package_spec']`). I am not sure how can I share the setting logic?